bin/vhost

```diff
@@ -174,10 +175,33 @@ fi
 #═════════════════════════════════════════════════════════════════════════════════════════════════
 if [ "$command" = "install" ]; then
   require_config_dir
+
   ip=127.0.0.1
-  hosts=$( jq -rM '.hosts | join(" ")' $config_file )
+  # Merge local config if any
+  if test -f "$local_config_file"; then
+    ptask "Using local config"
+    config=$(jq -s '.[0] * .[1]' $config_file $local_config_file)
+  else
+    config=$(cat $config_file)
+  fi
+  hosts=$(echo $config | jq -rM '.hosts | join(" ")')
+
+  if [[  "$(echo $config  | jq 'has("env")')" == "true" ]]; then
```

Comment:
Kind of too bad we need 2 branches for this, but it's a bit over my head so I'll take your word for it

Reply:
It's been a while since I wrote this code, but I'm assuming there was a reason...

Reply:
I figured it out. Without the conditional it would fail with:

`null (null) has no keys`

but it's easy to add a fallback with `// {}` so that it won't fail. Pushed.